src/Symfony/Bundle/FrameworkBundle/Tests/Command/AboutCommand/AboutCommandTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Command\AboutCommand;
+
+use Symfony\Bundle\FrameworkBundle\Command\AboutCommand;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Tests\Command\AboutCommand\Fixture\TestAppKernel;
+use Symfony\Bundle\FrameworkBundle\Tests\TestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem;
+
+class AboutCommandTest extends TestCase
+{
+    /** @var TestAppKernel */
+    private $kernel;
+    /** @var Filesystem */
+    private $fs;
+
+    protected function setUp(): void
+    {
+        $this->fs = new Filesystem();
+        $this->kernel = new TestAppKernel('test', true);
+        $this->fs->mkdir($this->kernel->getProjectDir());
+    }
+
+    protected function tearDown(): void
+    {
+        try {
+            $this->fs->remove($this->kernel->getProjectDir());
+        } catch (IOException $e) {
+        }
+    }
+
+    public function testAboutWithReadableFiles()
+    {
+        $this->fs->dumpFile($this->kernel->getCacheDir().'/readable_file', str_repeat('XXX', 1000000));
+        $this->fs->chmod($this->kernel->getCacheDir().'/readable_file', 0777);
+
+        $tester = $this->createCommandTester();
+        $ret = $tester->execute([]);
+
+        $this->assertEquals(0, $ret, 'Returns 0 in case of success');
+        $this->assertStringContainsString('Cache directory', $tester->getDisplay());
+        $this->assertStringContainsString('Log directory', $tester->getDisplay());
+
+        $this->fs->chmod($this->kernel->getCacheDir().'/readable_file', 0777);
+    }
+
+    public function testAboutWithUnreadableFiles()
+    {
+        // skip test on Windows; PHP can't easily set file as unreadable on Windows
+        if ('\\' === \DIRECTORY_SEPARATOR) {
+            $this->markTestSkipped('This test cannot run on Windows.');
```

Comment:
Why? We are using `is_readable` a lot in the code, I would be interested in understanding the issues on Windows.

Reply:
I based this test on `FilesystemTest`, see https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Filesystem/Tests/FilesystemTest.php#L44